packages/syft/src/syft/core/node/common/tables/bin_obj.py

```diff
@@ -0,0 +1,36 @@
+# stdlib
+from typing import Any
+
+# third party
+from sqlalchemy import Column
+from sqlalchemy import LargeBinary
+from sqlalchemy import String
+from sqlalchemy.orm.decl_api import DeclarativeMeta
+
+# syft absolute
+from syft import deserialize
+from syft import serialize
+
+
+def BinObject(TableBase: DeclarativeMeta) -> DeclarativeMeta:
+    """Initialize the BinObject Table and attached it to the TableBase"""
+
+    class BinObject(TableBase):  # type: ignore
+        __tablename__ = "bin_object"
+
+        id = Column(String(3072), primary_key=True)
+        binary = Column(LargeBinary(3072))
+        obj_name = Column(String(3072))
+
+        @property
+        def object(self) -> Any:
+            # storing DataMessage, we should unwrap
+            return deserialize(self.binary, from_bytes=True)  # TODO: techdebt fix
+
+        @object.setter
+        def object(self, value: Any) -> None:
+            # storing DataMessage, we should unwrap
+            self.binary = serialize(value, to_bytes=True)  # TODO: techdebt fix
+            # self.obj_name = type(value).__name__
+
+    return BinObject
```

Comment:
Isn't this quite duplicated work with the one in PyGrid?

Reply:
(this was resolved through an offline discussion between Tudor and myself. This is duplicate because we're in the process of moving this class from Grid -> Syft)